release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -99,6 +99,13 @@ For more information, see xref:../installing/cluster-capabilities.adoc#cluster-c
 ==== Root volume types parameter for {rh-openstack} is now available
 You can now specify one or more root volume types in {rh-openstack}, by using the `rootVolume.types` parameter. This parameter is available for both control plane and compute machines.
 
+[id="ocp-4-14-quickly-install-cluster-aws-local-zones"]
+==== Install a cluster quickly in AWS Local Zones
+For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to Local Zone locations. By using this installation route, the installation program automatically creates network resources and Local Zone subnets for each Local Zone that you defined in your configuration file. 
+
+For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Installing a cluster on AWS with worker nodes on AWS Local Zones]
```

Comment:
You pointed to a quickly section, but not described it in the naming. I would added between brackets the name of the section. WDYT?
```suggestion
For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Install a cluster quickly in AWS Local Zones]
```

Reply:
Good point. I added a more specific link. My original intention was to showcase the two routes, but RH AWS LZ users should already be familiar with the initial cluster installation route.